src/QsCompiler/CommandLineTool/Commands/Build.cs

```diff
@@ -177,18 +172,9 @@ private static IEnumerable<string> SplitCommandLineArguments(string commandLine)
         /// <summary>
         /// Builds the compilation for the Q# code or Q# snippet and referenced assemblies defined by the given options.
         /// Returns a suitable error code if one of the compilation or generation steps fails.
-        /// Throws an ArgumentNullException if any of the given arguments is null.
         /// </summary>
         public static int Run(BuildOptions options, ConsoleLogger logger)
         {
-            if (options == null)
```

Comment:
Don't we still need these checks for parameters on public facing methods? If I recall correctly, the fact that the method could be called from code that does not use nullable references meant that one could pass in a null as an argument to a public method. Has that been addressed?

Reply:
@bettinaheim and I believe that the checks are not worth it even for public methods. We recommend that users enable nullable reference types in their projects and avoid passing nulls to non-nullable parameters.